pkg/ebpf/bytecode/runtime/conntrack.go

```diff
@@ -0,0 +1,7 @@
+// Code generated by go generate; DO NOT EDIT.
+//go:build linux_bpf
+// +build linux_bpf
+
+package runtime
+
+var Conntrack = newAsset("conntrack.c", "7e5690072a127205273095970f65ca247d1a8328909ba5d102560c60615544d9")
```

Comment:
@akarpz the hash files in `pkg/ebpf/bytecode/runtime` should no longer be checked in. Please rebase/merge so you get the update `.gitignore`.